op-batcher/batcher/sync_actions.go

```diff
@@ -71,8 +71,8 @@ func computeSyncActions[T channelStatuser](
 	}
 
 	// PART 1: Initial checks on the sync status
-	if newSyncStatus.HeadL1 == (eth.L1BlockRef{}) {
-		m.Warn("empty sync status")
+	if safeL2 == (eth.L2BlockRef{}) || newSyncStatus.UnsafeL2 == (eth.L2BlockRef{}) {
+		m.Warn("empty L2BlockRef in sync status")
 		return syncActions{}, true
 	}
```

Comment:
wondering if we want to retain the previous check as a separate check and return a separate error message? We need all to be non-zero, unsafe, safe, and l1head.

Reply:
I put it back here and tweaked the error message. We log all of these fields anyway, so it will be possible to see which are zero and which are not.